src/api/zodClient.ts

```diff
@@ -23,10 +23,16 @@ const OptInSettings = z.object({
 const SDKTypeVisibilitySettings = z.object({
     enabledInFeatureSettings: z.boolean(),
 })
+const ObfuscationSettings = z.object({
+    enabled: z.boolean(),
+    required: z.boolean(),
+    key: z.array(z.number()),
```

Comment:
What is the `key` property used for? This doesn't seem to be added in your change to the API entity

Reply:
need to remove this